core/modules/comment/comment.entity.inc

```diff
@@ -435,4 +435,102 @@ class CommentStorageController extends EntityDatabaseStorageController {
         ->execute();
     }
   }
+
+  /**
+   * Implements EntityControllerInterface::buildContent().
+   */
+  public function buildContent(EntityInterface $comment, $view_mode = 'full', $langcode = NULL, $context = array()) {
+    if (!isset($langcode)) {
+      $langcode = $GLOBALS['language_content']->langcode;
```

Comment:
I've changed them. Actually still with a conditional because the `$langcode` could still be already set.

I noticed a couple of the previous format though but this is better.